truffle-config.js

```diff
@@ -76,7 +76,7 @@ module.exports = {
   compilers: {
     solc: {
       version: '0.5.17',
-      docker: true,
+      docker: false,
```

Comment:
Curious about this one. Is this required?

Reply:
config docs: https://trufflesuite.com/docs/truffle/reference/configuration/#solc
```
docker: <boolean>, // Use a version obtained through docker
```

with the bumped node version i am getting errors like this for some reason `Error: Error: spawnSync /bin/sh ENOBUFS`. turning this docker version off fixed it (as stackoverflow suggested). i am not sure if this is the right way to go.